tests/testsuite/registry.rs

```diff
@@ -2698,6 +2698,8 @@ fn http_requires_trailing_slash() {
         .run()
 }
 
+// Run this test only on debug build.
```

Comment:
Could you clarify this comment, maybe something like
```
// Limit the test to debug builds so that `__CARGO_TEST_MAX_UNPACK_SIZE` will take affect
```

Reply:
Good suggestion. Updated (and #11091 as well).